src/ReverseProxy/Abstractions/SessionAffinityConfig.cs

```diff
@@ -33,7 +39,7 @@ public sealed record SessionAffinityConfig
         /// This value should be unique across clusters to avoid affinity conflicts.
         /// https://github.com/microsoft/reverse-proxy/issues/976
         /// </summary>
-        public string? AffinityKeyName { get; init; }
+        public string AffinityKeyName { get; init; }
```

Comment:
Update the comment to indicate this is required.
```suggestion
        public string AffinityKeyName { get; init; } = default!;
```

Reply:
Done